nixos/modules/tasks/network-interfaces.nix

```diff
@@ -565,6 +582,7 @@ in
             example = 100;
             type = types.nullOr types.int;
             description = ''
+              DEPRECATED, use `driverOptions`.
```

Comment:
Instead of writing "DEPRECATED" in the description, I'd either pass down miimon/lacp_rate/mode/... to `driverOptions` in the definition or properly deprecate/remove it via [`rename.nix`](https://github.com/NixOS/nixpkgs/blob/9f334c2732e4d8606ccb09b14893aad5c37bb9da/nixos/modules/rename.nix).